<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="1" total="8" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="SuiteOne" duration-ms="127" started-at="2016-12-29T06:43:30Z" finished-at="2016-12-29T06:43:30Z">
    <groups>
    </groups>
    <test name="SuiteOneCaseOne" duration-ms="24" started-at="2016-12-29T06:43:30Z" finished-at="2016-12-29T06:43:30Z">
      <class name="com.stta.SuiteOne.SuiteOneCaseOne">
        <test-method status="SKIP" signature="checkCaseToRun()[pri:0, instance:com.stta.SuiteOne.SuiteOneCaseOne@192b07fd]" name="checkCaseToRun" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="SKIP" signature="SuiteOneCaseOneTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.stta.SuiteOne.SuiteOneCaseOne@192b07fd]" name="SuiteOneCaseOneTest" duration-ms="0" started-at="2016-12-29T00:43:30Z" data-provider="SuiteOneCaseOneData" finished-at="2016-12-29T00:43:30Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:110)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:436)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1253)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.NullPointerException
	at com.stta.utility.SuiteUtility.GetTestDataUtility(SuiteUtility.java:27)
	at com.stta.SuiteOne.SuiteOneCaseOne.SuiteOneCaseOneData(SuiteOneCaseOne.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:51)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:41)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteOneCaseOneTest -->
        <test-method status="SKIP" signature="closeBrowser()[pri:0, instance:com.stta.SuiteOne.SuiteOneCaseOne@192b07fd]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.stta.SuiteOne.SuiteOneCaseOne -->
    </test> <!-- SuiteOneCaseOne -->
    <test name="SuiteOneCaseTwo" duration-ms="11" started-at="2016-12-29T06:43:30Z" finished-at="2016-12-29T06:43:30Z">
      <class name="com.stta.SuiteOne.SuiteOneCaseTwo">
        <test-method status="SKIP" signature="checkSuiteToRun()[pri:0, instance:com.stta.SuiteOne.SuiteOneCaseTwo@2286778]" name="checkSuiteToRun" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[SuiteOne's SuiteToRun Flag Is 'N' Or Blank. So Skipping Execution Of SuiteOne]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: SuiteOne's SuiteToRun Flag Is 'N' Or Blank. So Skipping Execution Of SuiteOne
	at com.stta.SuiteOne.SuiteOneBase.checkSuiteToRun(SuiteOneBase.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:307)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSuiteToRun -->
        <test-method status="SKIP" signature="checkCaseToRun()[pri:0, instance:com.stta.SuiteOne.SuiteOneCaseTwo@2286778]" name="checkCaseToRun" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="SKIP" signature="SuiteOneCaseTwoTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.stta.SuiteOne.SuiteOneCaseTwo@2286778]" name="SuiteOneCaseTwoTest" duration-ms="0" started-at="2016-12-29T00:43:30Z" data-provider="SuiteOneCaseTwoData" finished-at="2016-12-29T00:43:30Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:110)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:436)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1253)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.NullPointerException
	at com.stta.utility.SuiteUtility.GetTestDataUtility(SuiteUtility.java:27)
	at com.stta.SuiteOne.SuiteOneCaseTwo.SuiteOneCaseTwoData(SuiteOneCaseTwo.java:164)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:51)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:41)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteOneCaseTwoTest -->
        <test-method status="SKIP" signature="closeBrowser()[pri:0, instance:com.stta.SuiteOne.SuiteOneCaseTwo@2286778]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.stta.SuiteOne.SuiteOneCaseTwo -->
    </test> <!-- SuiteOneCaseTwo -->
  </suite> <!-- SuiteOne -->
  <suite name="SuiteThree" duration-ms="26537" started-at="2016-12-29T06:43:31Z" finished-at="2016-12-29T06:43:58Z">
    <groups>
    </groups>
    <test name="SuiteThree_TestCase1" duration-ms="436" started-at="2016-12-29T06:43:31Z" finished-at="2016-12-29T06:43:32Z">
      <class name="com.stta.SuiteThree.SuiteThreeCaseOne">
        <test-method status="PASS" signature="checkCaseToRun()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@369f73a2]" name="checkCaseToRun" is-config="true" duration-ms="15" started-at="2016-12-29T00:43:31Z" finished-at="2016-12-29T00:43:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="PASS" signature="initializeTestProperties()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@369f73a2]" name="initializeTestProperties" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:31Z" finished-at="2016-12-29T00:43:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeTestProperties -->
        <test-method status="PASS" signature="TC_01_TestCase(int, java.util.Map)[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@369f73a2]" name="TC_01_TestCase" duration-ms="128" started-at="2016-12-29T00:43:31Z" data-provider="setData" finished-at="2016-12-29T00:43:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[{TestNGTestName=SuiteThree_TestCase1, Iteration=, Pass/Fail/Skip=PASS, TestCaseName=TC_01_TestCase, Data 2 Column=5, DataToRun=y, Data 3 Column=7, Expected Result=21, Data 1 Column=9}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_01_TestCase -->
        <test-method status="PASS" signature="closeTest()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@369f73a2]" name="closeTest" is-config="true" duration-ms="74" started-at="2016-12-29T00:43:31Z" finished-at="2016-12-29T00:43:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeTest -->
        <test-method status="PASS" signature="initializeTestProperties()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@369f73a2]" name="initializeTestProperties" is-config="true" duration-ms="1" started-at="2016-12-29T00:43:31Z" finished-at="2016-12-29T00:43:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeTestProperties -->
        <test-method status="PASS" signature="TC_01_TestCase(int, java.util.Map)[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@369f73a2]" name="TC_01_TestCase" duration-ms="122" started-at="2016-12-29T00:43:31Z" data-provider="setData" finished-at="2016-12-29T00:43:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[{TestNGTestName=SuiteThree_TestCase1, Iteration=, Pass/Fail/Skip=SKIP, TestCaseName=TC_01_TestCase, Data 2 Column=7, DataToRun=y, Data 3 Column=3, Expected Result=13, Data 1 Column=3}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_01_TestCase -->
        <test-method status="PASS" signature="closeTest()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@369f73a2]" name="closeTest" is-config="true" duration-ms="70" started-at="2016-12-29T00:43:32Z" finished-at="2016-12-29T00:43:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeTest -->
      </class> <!-- com.stta.SuiteThree.SuiteThreeCaseOne -->
    </test> <!-- SuiteThree_TestCase1 -->
    <test name="SuiteThree_TestCase2" duration-ms="223" started-at="2016-12-29T06:43:32Z" finished-at="2016-12-29T06:43:32Z">
      <class name="com.stta.SuiteThree.SuiteThreeCaseOne">
        <test-method status="PASS" signature="checkCaseToRun()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@65e2dbf3]" name="checkCaseToRun" is-config="true" duration-ms="21" started-at="2016-12-29T00:43:32Z" finished-at="2016-12-29T00:43:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="PASS" signature="initializeTestProperties()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@65e2dbf3]" name="initializeTestProperties" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:32Z" finished-at="2016-12-29T00:43:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeTestProperties -->
        <test-method status="PASS" signature="TC_02_TestCase(int, java.util.Map)[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@65e2dbf3]" name="TC_02_TestCase" duration-ms="124" started-at="2016-12-29T00:43:32Z" data-provider="setData" finished-at="2016-12-29T00:43:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[{TestNGTestName=SuiteThree_TestCase2, Iteration=, Pass/Fail/Skip=, TestCaseName=TC_02_TestCase, Data 2 Column=5, DataToRun=y, Data 3 Column=7, Expected Result=21, Data 1 Column=9}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_02_TestCase -->
        <test-method status="PASS" signature="closeTest()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseOne@65e2dbf3]" name="closeTest" is-config="true" duration-ms="74" started-at="2016-12-29T00:43:32Z" finished-at="2016-12-29T00:43:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeTest -->
      </class> <!-- com.stta.SuiteThree.SuiteThreeCaseOne -->
    </test> <!-- SuiteThree_TestCase2 -->
    <test name="SuiteThree_TestCase3" duration-ms="25742" started-at="2016-12-29T06:43:32Z" finished-at="2016-12-29T06:43:58Z">
      <class name="com.stta.SuiteThree.SuiteThreeCaseTwo">
        <test-method status="PASS" signature="checkSuiteToRun()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseTwo@5bc79255]" name="checkSuiteToRun" is-config="true" duration-ms="607" started-at="2016-12-29T00:43:31Z" finished-at="2016-12-29T00:43:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSuiteToRun -->
        <test-method status="PASS" signature="checkCaseToRun()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseTwo@5bc79255]" name="checkCaseToRun" is-config="true" duration-ms="8889" started-at="2016-12-29T00:43:32Z" finished-at="2016-12-29T00:43:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="FAIL" signature="TC_03_TestCase(int, java.util.Map)[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseTwo@5bc79255]" name="TC_03_TestCase" duration-ms="16742" started-at="2016-12-29T00:43:41Z" data-provider="setData" finished-at="2016-12-29T00:43:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[{TestNGTestName=SuiteThree_TestCase3, Iteration=, Pass/Fail/Skip=, TestCaseName=TC_03_TestCase, Data 2 Column=7, DataToRun=y, Data 3 Column=3, Expected Result=13, Data 1 Column=3}]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: {"method":"id","selector":"Resultbox"}
Command duration or timeout: 10.09 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'LTPUSNY2127380', ip: '192.168.91.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_05'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: f52483b4-5642-460f-831e-fea7cc96a694
*** Element info: {Using=id, value=Resultbox}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"id","selector":"Resultbox"}
Command duration or timeout: 10.09 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'LTPUSNY2127380', ip: '192.168.91.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_05'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: f52483b4-5642-460f-831e-fea7cc96a694
*** Element info: {Using=id, value=Resultbox}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:413)
	at org.openqa.selenium.By$ById.findElement(By.java:218)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy11.isDisplayed(Unknown Source)
	at com.stta.TestSuiteBase.SuiteBase.isWebElementDisplayed(SuiteBase.java:269)
	at com.stta.PageObject.CalculatePage.isLoaded(CalculatePage.java:53)
	at org.openqa.selenium.support.ui.LoadableComponent.get(LoadableComponent.java:46)
	at com.stta.SuiteThree.SuiteThreeCaseTwo.TC_03_TestCase(SuiteThreeCaseTwo.java:107)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"id","selector":"Resultbox"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'LTPUSNY2127380', ip: '192.168.91.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_05'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/BANTI~1.CHO/AppData/Local/Temp/anonymous340512094151840531webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/BANTI~1.CHO/AppData/Local/Temp/anonymous340512094151840531webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_03_TestCase -->
        <test-method status="PASS" signature="initializeTestProperties()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseTwo@5bc79255]" name="initializeTestProperties" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:41Z" finished-at="2016-12-29T00:43:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeTestProperties -->
        <test-method status="PASS" signature="closeTest()[pri:0, instance:com.stta.SuiteThree.SuiteThreeCaseTwo@5bc79255]" name="closeTest" is-config="true" duration-ms="100" started-at="2016-12-29T00:43:58Z" finished-at="2016-12-29T00:43:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeTest -->
      </class> <!-- com.stta.SuiteThree.SuiteThreeCaseTwo -->
    </test> <!-- SuiteThree_TestCase3 -->
  </suite> <!-- SuiteThree -->
  <suite name="Calc Suite" duration-ms="0" started-at="2016-12-29T06:43:59Z" finished-at="2016-12-29T06:43:59Z">
    <groups>
    </groups>
  </suite> <!-- Calc Suite -->
  <suite name="SuiteTwo" duration-ms="116" started-at="2016-12-29T06:43:30Z" finished-at="2016-12-29T06:43:31Z">
    <groups>
    </groups>
    <test name="SuiteTwoCaseOne" duration-ms="12" started-at="2016-12-29T06:43:30Z" finished-at="2016-12-29T06:43:30Z">
      <class name="com.stta.SuiteTwo.SuiteTwoCaseOne">
        <test-method status="SKIP" signature="checkCaseToRun()[pri:0, instance:com.stta.SuiteTwo.SuiteTwoCaseOne@6b09bb57]" name="checkCaseToRun" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="SKIP" signature="SuiteTwoCaseOneTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.stta.SuiteTwo.SuiteTwoCaseOne@6b09bb57]" name="SuiteTwoCaseOneTest" duration-ms="0" started-at="2016-12-29T00:43:30Z" data-provider="SuiteTwoCaseOneData" finished-at="2016-12-29T00:43:30Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:110)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:436)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1253)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.NullPointerException
	at com.stta.utility.SuiteUtility.GetTestDataUtility(SuiteUtility.java:27)
	at com.stta.SuiteTwo.SuiteTwoCaseOne.SuiteTwoCaseOneData(SuiteTwoCaseOne.java:153)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:51)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:41)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteTwoCaseOneTest -->
        <test-method status="SKIP" signature="closeBrowser()[pri:0, instance:com.stta.SuiteTwo.SuiteTwoCaseOne@6b09bb57]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.stta.SuiteTwo.SuiteTwoCaseOne -->
    </test> <!-- SuiteTwoCaseOne -->
    <test name="SuiteTwoCaseTwo" duration-ms="9" started-at="2016-12-29T06:43:31Z" finished-at="2016-12-29T06:43:31Z">
      <class name="com.stta.SuiteTwo.SuiteTwoCaseTwo">
        <test-method status="SKIP" signature="checkSuiteToRun()[pri:0, instance:com.stta.SuiteTwo.SuiteTwoCaseTwo@335eadca]" name="checkSuiteToRun" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:30Z" finished-at="2016-12-29T00:43:30Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[SuiteTwo's SuiteToRun Flag Is 'N' Or Blank. So Skipping Execution Of SuiteTwo]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: SuiteTwo's SuiteToRun Flag Is 'N' Or Blank. So Skipping Execution Of SuiteTwo
	at com.stta.SuiteTwo.SuiteTwoBase.checkSuiteToRun(SuiteTwoBase.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:307)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSuiteToRun -->
        <test-method status="SKIP" signature="checkCaseToRun()[pri:0, instance:com.stta.SuiteTwo.SuiteTwoCaseTwo@335eadca]" name="checkCaseToRun" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:31Z" finished-at="2016-12-29T00:43:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCaseToRun -->
        <test-method status="SKIP" signature="SuiteTwoCaseTwoTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.stta.SuiteTwo.SuiteTwoCaseTwo@335eadca]" name="SuiteTwoCaseTwoTest" duration-ms="0" started-at="2016-12-29T00:43:31Z" data-provider="SuiteTwoCaseTwoData" finished-at="2016-12-29T00:43:31Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:110)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:436)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1253)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.NullPointerException
	at com.stta.utility.SuiteUtility.GetTestDataUtility(SuiteUtility.java:27)
	at com.stta.SuiteTwo.SuiteTwoCaseTwo.SuiteTwoCaseTwoData(SuiteTwoCaseTwo.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:51)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:41)
	... 23 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteTwoCaseTwoTest -->
        <test-method status="SKIP" signature="closeBrowser()[pri:0, instance:com.stta.SuiteTwo.SuiteTwoCaseTwo@335eadca]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2016-12-29T00:43:31Z" finished-at="2016-12-29T00:43:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.stta.SuiteTwo.SuiteTwoCaseTwo -->
    </test> <!-- SuiteTwoCaseTwo -->
  </suite> <!-- SuiteTwo -->
</testng-results>
